### PR TITLE
Using debug action in server code step fixed (closes #4245)

### LINKED ISSUE #4245: Error raises during testrun server code step if it contains debug action
RAW API test example
```
{
    "fixtures": [
        {
            "name": "test test",
            "pageUrl": "http://localhost/site",
            "tests": [
                {
                    "name": "New Test",
                    "commands": [
                        {
                            "type": "execute-async-expression",
                            "studio": {},
                            "callsite": "0",
                            "expression": "await t\r\n    .debug();"
                        }
                    ]
                }
            ]
        }
    ]
}
```

The error:
```
An unhandled error occurred in a step with custom JS code:

ENOENT: no such file or directory, open 'D:\GitHub\testcafe-studio\[JS code]
  await t
      .debug();

await t
.debug()
at 646:18
```

### PR DESCRIPTION
**Pull request by helen-dikareva**

### CODE REVIEW

File: src/notifications/debug-logger.js [hunk 1]
```diff
@@ -62,19 +63,24 @@ export default {
 
         // NOTE: Raw API does not have callsite.
```
Comment (AlexKamaev) [inline src/notifications/debug-logger.js]: is this note actual?

File: src/notifications/debug-logger.js [hunk 2]
```diff
@@ -62,19 +63,24 @@ export default {
...
         const hasCallsite = callsite && callsite.renderSync;
```
Comment (AlexKamaev) [inline src/notifications/debug-logger.js]: we have multiple callsite existence checks in the code, should we extract them into the `renderCallsiteSync` method?

File: src/notifications/debug-logger.js (hunk 1 above)
Comment (helen-dikareva) [reply src/notifications/debug-logger.js]: yep

File: src/notifications/debug-logger.js (hunk 2 above)
Comment (helen-dikareva) [reply src/notifications/debug-logger.js]: hm, ok

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Pull request merged**